spec/ruby_terraform/commands/clean_spec.rb

```diff
@@ -1,11 +1,20 @@
 require 'spec_helper'
 
 describe RubyTerraform::Commands::Clean do
+  before(:each) do
+    RubyTerraform.configure do |config|
+      @logger_mock = config.logger
+    end
+  end
+
```

Comment:
To maintain test isolation, there should really be a `RubyTerraform.reset!` in an after each block.

Reply:
True, good catch. I forgot.

Reply:
Fixed